x-pack/plugins/apm/public/tutorial/config_agent/opentelemetry_instructions.tsx

```diff
@@ -0,0 +1,152 @@
+      setting: 'OTEL_EXPORTER_OTLP_HEADERS',
+      value: `"Authorization=Bearer ${
+        secretToken ? secretToken : '<secret-token>'
+      }"`,
+    },
+    {
+      setting: 'OTEL_METRICS_EXPORTER',
+      value: 'otlp',
+      notes: 'Enable metrics when supported by your OpenTelemtry client.',
+    },
+    {
+      setting: 'OTEL_LOGS_EXPORTER',
+      value: 'otlp',
+      notes: 'Enable logs when supported by your OpenTelemetry client',
+    },
+    {
+      setting: 'OTEL_RESOURCE_ATTRIBUTES',
+      value:
+        'service.name=<app-name>,service.version=<app-version>,deployment.environment=production',
+    },
+  ];
+
+  const columns: Array<EuiBasicTableColumn<ValuesType<typeof items>>> = [
+    {
+      field: 'setting',
+      name: i18n.translate(
+        'xpack.apm.tutorial.config_otel.column.configSettings',
+        {
+          defaultMessage: 'Configuration setting (1)',
+        }
+      ),
+    },
+    {
+      field: 'value',
+      name: i18n.translate(
+        'xpack.apm.tutorial.config_otel.column.configValue',
+        {
+          defaultMessage: 'Configuration value',
+        }
+      ),
+      render: (_, { value }) => <EuiText color="accent">{value}</EuiText>,
+    },
+    {
+      field: 'notes',
+      name: i18n.translate('xpack.apm.tutorial.config_otel.column.notes', {
+        defaultMessage: 'Notes',
+      }),
+    },
+  ];
+
+  return (
+    <>
+      <EuiBasicTable
+        items={items}
+        columns={columns}
+        data-test-subj="otel-instructions-table"
+      />
+      <EuiSpacer size="m" />
+      <EuiText size="xs" color="subdued">
+        <FormattedMessage
+          id="xpack.apm.tutorial.config_otel.description1"
+          defaultMessage="(1) OpenTelemetry agents and SDKs must support the {otelExporterOtlpEndpoint}, {otelExporterOtlpHeaders}, and {otelResourceAttributes}; some unstable components may not yet comply with this requirement."
+          values={{
+            otelExporterOtlpEndpoint: (
+              <EuiLink
+                target="_blank"
+                href="https://github.com/open-telemetry/opentelemetry-specification/blob/v1.10.0/specification/protocol/exporter.md"
+              >
+                OTEL_EXPORTER_OTLP_ENDPOINT
+              </EuiLink>
+            ),
+            otelExporterOtlpHeaders: (
+              <EuiLink
+                target="_blank"
+                href="https://github.com/open-telemetry/opentelemetry-specification/blob/v1.10.0/specification/protocol/exporter.md"
+              >
+                OTEL_EXPORTER_OTLP_HEADERS
+              </EuiLink>
+            ),
+            otelResourceAttributes: (
+              <EuiLink
+                target="_blank"
+                href="https://github.com/open-telemetry/opentelemetry-specification/blob/v1.10.0/specification/resource/sdk.md"
+              >
+                OTEL_RESOURCE_ATTRIBUTES
+              </EuiLink>
+            ),
+          }}
+        />
+        <br />
```

Comment:
Can't you use EuiSpacer instead?
```suggestion
        <EuiSpacer />
```